apps/frontend/src/libs/components/courses/courses.tsx

```diff
@@ -1,30 +1,61 @@
 import { type CourseDto } from "~/modules/courses/courses.js";
-import { type AddCourseRequestDto } from "~/modules/user-courses/user-courses.js";
+import {
+	type AddCourseRequestDto,
+	type UserCourseResponseDto,
+} from "~/modules/user-courses/user-courses.js";
 
 import { Course } from "../course/course.js";
 import styles from "./styles.module.css";
 
 type Properties = {
-	courses: CourseDto[];
+	courses?: CourseDto[];
```

Comment:
```suggestion
	courses: CourseDto[];
```
for courses should be only one key in props

Reply:
Did I understand [that](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-trackmates/pull/282#discussion_r1512752866) incorrectly?